README.md

```diff
@@ -72,8 +72,8 @@ Other features of ArangoDB include:
   durability or more performance
 - **Schema-free schemata** let you combine the space efficiency of MySQL with the
   performance power of NoSQL
-- Free **index choice**: use the correct index for your problem, be it a skiplist 
-  or a fulltext search
+- Free **index choice**: use the correct index for your problem, be it a skip list 
+  or a full text search
```

Comment:
The index types are called "skiplist" and "fulltext" internally and it's actually easier to grep for these terms without false positives if they are written as single word. Not sure if we want to change this. @sleto-it?